frameworks/react/src/AdvancedVideo.tsx

```diff
@@ -116,15 +123,45 @@ class AdvancedVideo extends React.Component <VideoProps> {
     };
   }
 
+  /**
+   * Attach both this.videoRef and props.innerRef as ref to the given element
+   * @param element - the element to attach a ref to
+   */
+  attachRef(element: HTMLVideoElement) {
+    this.videoRef.current = element;
```

Comment:
Why do we need both videoRef and innerRef?

Reply:
innerRef is user controlled, videoRef is internal

Reply:
Right. We might want to name them better if that's the case.
I understand `videoRef` was the existing one before the innerRef addition, 
Do you have a naming suggestion to make this separation clearer?